networks/rpc/http_datadog.go

```diff
@@ -0,0 +1,145 @@
+package rpc
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	httptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type DatadogTracer struct {
+	Tags           []string
+	Service        string
+	KlaytnResponse bool
+}
+
+func newDatadogTracer() *DatadogTracer {
+	ddTraceEnabled := os.Getenv("DD_TRACE_ENABLED")
+	if strings.ToLower(ddTraceEnabled) != "true" {
+		return nil
+	}
+
+	tracer.Start()
+
+	tags := strings.Split(os.Getenv("DD_TRACE_HEADER_TAGS"), ",")
+	service := os.Getenv("DD_SERVICE")
+	klaytnResponse := strings.ToLower(os.Getenv("DD_KLAYTN_RPC_RESPONSE")) == "true"
```

Comment:
Just out of curiosity. Is there any reason to use "true" instead of 1?

Reply:
All other datadog related environment variables have true/false values.
So I used true/false to go similar.

ex)
https://github.com/DataDog/dd-trace-go/blob/main/ddtrace/tracer/option.go#L236 
https://ddtrace.readthedocs.io/en/stable/configuration.html

Reply:
It seems that your reference is using a boolean value, but "true" is a string.

Reply:
Oh. I know what you're saying. 
I Changed to use parseBool instead of string compare. Would this be okay?

Reply:
Thanks!